tests/unit/lms/validation/authentication/_oauth_test.py

```diff
@@ -234,14 +234,14 @@ def test_valid(self, json_data):
                     "access_token": "test_access_token",
                     "expires_in": 0,
                 },
-                {"expires_in": ["expires_in must be greater than 0"]},
+                {"expires_in": ["Must be greater than or equal to 1."]},
```

Comment:
Marshmallow's builtin error message is slightly different from the manual one that we were using